venv/lib/python3.12/site-packages/PIL/ImageDraw.py

```
@@ -0,0 +1,1218 @@
+            try:
+                mask, offset = font.getmask2(  # type: ignore[union-attr,misc]
+                    text,
+                    mode,
+                    direction=direction,
+                    features=features,
+                    language=language,
+                    stroke_width=stroke_width,
+                    anchor=anchor,
+                    ink=ink,
+                    start=start,
+                    *args,
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)